pages/linux/xeyes.md

```diff
@@ -0,0 +1,11 @@
+# xeyes
+
+> Display eyes on the screen that follow the mouese cursor
+
+- Launch xeyes on local display:
+
+`xeyes`
+
+- Launch xeyes on a remote display:
+
+`xeyes -display {{remote.host}}:0.0`
```

Comment:
Ah ok, then they should be in tokens too ! 
- {{remote_host}}:{{0}}.{{0}}

And mention that `Launch xeyes on a remote display with terminal 0 and screen 0`.